docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -144,6 +144,10 @@ that is printed out.
 
 For the exact list of named groups ``@linter`` recognizes, see the `API
 documentation <https://api.coala.io/en/latest/>`__.
+Pylint uses 0-based column convention and 1-based line convention.
```

Comment:
Add a reminder here is not very useful IMO. And the reminder does not bring any new information.
See https://github.com/coala/coala/issues/5297#issuecomment-440341156